### ISSUE TITLE
[BFCL] Multi-turn query missing info & incorrect label

### ISSUE THREAD

**Issue by mingzhu0527** [ShishirPatil/gorilla#771]
**Describe the issue**
Hi, recently we reviewed some multi-turn data from BFCL-v3, and noticed a few cases where there's not enough information in the query to complete the task, or the groundtruth is incorrect.

**ID datapoint**
ID | Query | GroundTruth Answer | Comment
-- | -- | -- | --
multi_turn_base_1 | I am alex. Check if the current directory is under my name and list all the visible and hidden contents in the current directory now, please. | cd(folder='workspace')ls(a=True) | 'workspace' not mentioned in query
multi_turn_base_2 | Go into document folder and Could you draft up a create a document titled 'TeamNotes.txt' for keeping track of all the fresh ideas? | cd(folder='documents')touch(file_name='TeamNotes.txt') | Query use 'document' but ground
multi_turn_base_3 | As part of my latest photography project, I need to gather files that have 'test' in their name from a specific folder in my current directory. Could you help me locate those? | find(path='.',name='test') | Query missing info
multi_turn_base_4 | Could you kindly place the report in the /tmp directory or show me the list of files in tmp if already there? | pwd()ls() | GroundTruth error
multi_turn_base_11 | Display all the available files located within the '/temp' directory using the terminal interface. | pwd()ls(a=True) | GroundTruth error
multi_turn_base_18 | Copy the txt contents of the 'Quarter1_Reports' directory and place it in a new directory naming it 'Archived_Quarter1. | mkdir(dir_name='Archived_Quarter1')cp(source='report1.txt',destination='Archived_Quarter1')cp(source='report2.txt',destination='Archived_Quarter1')cp(source='History101.txt',destination='Archived_Quarter1')cp(source='History202.txt',destination='Archived_Quarter1') | GroundTruth Error

**Comment by mingzhu0527** [comment]
Thank you for your reply! 
6. multi_turn_base_18: The groundtruth error I'm talking about is that, the query says "Copy the txt contents of the '**Quarter1_Reports**' directory", but the groundtruth did not first navigate to Quarter1_Reports or check what is the current directory. 
You mentioned that "Our current working directory is Quarter1_Reports", but this information is not included in either the query or the system prompt, and because of which the model could not figure out. I wonder if it is supposed to be this way.

**Comment by Fanjia-Yan** [comment]
Yes, the **Quarter1_Reports** directory location is nowhere mentioned in the provided context. However, the model is able to obtain the information via `ls` or `pwd` to collect this layer of information if deemed necessary. We want the model to explore but we don't judge model's exploration or failure attempt, which is why we don't include this in ground truth.

We judge a data entry accuracy by 2 criteria:
1. Final state exact match
2. Minimal function calling steps to achieve all user queries(as annotated in ground truth)
There are steps such as `pwd` which is an exploratory step and is not required to complete the task, and therefore we do not include in ground truth. 

Thanks for bringing this up and sorry for the late reply. We will also document this procedure in a clearer manner.
